### PR TITLE
Minor code edits

### PR DESCRIPTION
**Pull request by dcol2804**
Code fixes for tutorials - qmd files were the only ones edited and involved edits to custom R code, traits.build functions that have since been updated and field updated names and text.

### CODE REVIEW

**Comment by dcol2804** [comment]
Great! I approve myself to merge this branch

File: tutorial_dataset_2.qmd [hunk 1]
```diff
@@ -407,8 +407,10 @@ custom_R_code: '
     mutate(
       across(c("TRAIT Leaf Dry Mass UNITS g"), ~na_if(.x,0))
     ) %>%
-    group_by(name_original) %>% #column with taxon name
-      across(c("TRAIT Growth Form CATEGORICAL EP epiphyte (mistletoe) F fern G grass H herb S shrub T tree V vine"), replace_duplicates_with_NA) %>%
+    group_by(name_original) %>%
```
Comment (ehwenk) [inline tutorial_dataset_2.qmd:410]: I had the note so people realised that `name_original` was the input column name, not an output column - did you find it not useful or confusing how I'd annotated it?

File: tutorial_dataset_4.qmd [hunk 2]
```diff
@@ -121,17 +121,15 @@ References
...
 ```
 
-You would now need to look up each of these references in the reference section of the manuscript and use Google Scholar (or another reference resource) to look up the doi for each reference.\
-
-To add additional references, you need to add an argument to the `metadata_add_source_doi` function:\
+You would now need to look up each of these references in the reference section of the manuscript and use Google Scholar (or another reference resource) to look up the doi for each reference.\To add additional references, you need to add an argument to the `metadata_add_source_doi` function:\
 
 ```{r, eval=FALSE}
-metadata_add_source_doi(dataset_id = "tutorial_dataset_4", doi = "10.1071/bt9960249", type = "secondary")
+metadata_add_source_doi(dataset_id = "tutorial_dataset_4", doi = "10.1071/bt9960249", type = "original_01")
```
Comment (ehwenk) [inline tutorial_dataset_4.qmd:127]: Does that work? I thought the function only accepted types "primary" and "secondary".

**Review by ehwenk — APPROVED** [review]
Thank you for catching these errors! 

I'll separately change all the`build_setup_pipeline` mentions to specify that one needs to assign a "name" to the database - or use the default.

**Pull request merged**